pandas/tests/frame/methods/test_convert_dtypes.py

```diff
@@ -134,3 +134,13 @@ def test_pyarrow_engine_lines_false(self):
         )
         with pytest.raises(ValueError, match=msg):
             df.convert_dtypes(dtype_backend="numpy")
+
+    @pytest.mark.xfail(
+        reason="The pyarrow doesn't currentely support rounding of large integers",
+    )
+    def test_pyarrow_dtype_backend_for_int64(self):
+        with tm.ensure_clean("__tmp_to_csv_from_csv3__") as path:
+            # GH 52827
+            pytest.importorskip("pyarrow")
+            pd.DataFrame({"x": ["1372636858620000589"]}).to_csv(path, index=False)
```

Comment:
Please avoid roundtripping through csv

Reply:
Thanks for the feedback.

Updated - checked locally and it throws the right error:
```pyarrow.lib.ArrowInvalid: Integer value 1372636858620000589 not in range: -9007199254740992 to 9007199254740992```